<ui version="4.0" >
 <class>ScreenInputDialog</class>
 <widget class="QDialog" name="ScreenInputDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>210</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Add a Screen</string>
  </property>
  <widget class="QPushButton" name="btnCancel" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>180</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>180</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>391</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title" >
    <string>Screen</string>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>146</height>
     </rect>
    </property>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="3" column="0" >
      <widget class="QLineEdit" name="cScreenBox" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QRadioButton" name="rDefault" >
       <property name="text" >
        <string>&amp;Default</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QComboBox" name="dScreenBox" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QRadioButton" name="rCustom" >
       <property name="text" >
        <string>&amp;Custom</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Screen text:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="screenText" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
